contracts/PillarStaking.sol

```diff
@@ -155,24 +182,21 @@ contract PillarStaking is ReentrancyGuard, Ownable {
         return stakeholderData[account].stakedAmount;
     }
 
+    function getRewardAmountForAccount(address account)
+        public
+        view
+        returns (uint256 amount)
+    {
+        return stakeholderData[account].rewardAmount;
+    }
+
     /* ========== RESTRICTED FUNCTIONS ========== */
 
-    function depositRewards(uint256 _amount) external onlyOwner {
+    function setRewards(uint256 _amount) external onlyOwner {
         if (_amount == 0) revert RewardsCannotBeZero();
         rewardToken.safeTransferFrom(msg.sender, address(this), _amount);
-        rewards = _amount;
-        emit RewardsDeposited(_amount);
-    }
-
-    function calculateRewardAllocation(address _staker)
-        internal
-        view
-        returns (uint256)
-    {
-        uint256 stakeAmount = getStakedAmountForAccount(_staker);
-        uint256 stakedPercentage = ((stakeAmount * BPS) / totalStaked);
-        uint256 reward = ((stakedPercentage * rewards) / BPS);
-        return reward;
+        rewards += _amount;
```

Comment:
sorry, my previous comment was ambiguous. I meant that the `depositRewards` implies that here should be `rewards += _amount`, but if we're renaming this function to `setRewards` then it's better to write `rewards = _amount` imo

Reply:
Ok i'll change the fn name back to depositRewards as its more descriptive of what the fn does.

Reply:
Done